wallet/models.py

```diff
@@ -13,14 +17,21 @@ def __str__(self):
         return self.name
 
 
+@receiver(pre_save, sender=Category)
+def validate_unique(sender, instance, **kwargs):
```

Comment:
It's better to use a unique constraint for this.

Reply:
I know that. But the owner field is nullable, UniqueTogeter does not work for that. For instance, you can have multiple records like {'owner': None, 'name': 'c'}